packages/scaffold-ui/src/views/w3m-send-confirmed-view/index.ts

```diff
@@ -0,0 +1,61 @@
+import { LitElement, html } from 'lit'
+
+import { ModalController } from '@reown/appkit-controllers'
+import { customElement } from '@reown/appkit-ui'
+import '@reown/appkit-ui/wui-button'
+import '@reown/appkit-ui/wui-flex'
+import '@reown/appkit-ui/wui-icon'
+import '@reown/appkit-ui/wui-text'
+
+import styles from './styles.js'
+
+@customElement('w3m-send-confirmed-view')
+export class W3mSendConfirmedView extends LitElement {
+  public static override styles = styles
+
+  // -- Members ------------------------------------------- //
+  private unsubscribe: (() => void)[] = []
+
+  public constructor() {
+    super()
+    this.unsubscribe.push(...[])
+  }
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    return html`
+      <wui-flex
+        flexDirection="column"
+        alignItems="center"
+        gap="4"
+        .padding="${['1', '3', '4', '3'] as const}"
+      >
+        <wui-flex justifyContent="center" alignItems="center" class="icon-box">
+          <wui-icon size="xxl" color="success" name="checkmark"></wui-icon>
+        </wui-flex>
+
+        <wui-text variant="h6-medium" color="primary">You successfully sent asset</wui-text>
```

Comment:
Can we use the asset's name here? 
Also in case we don't have it, typo? `You have successfully sent the asset` (I'd check w sam)